src/llmq/instantsend.h

```diff
@@ -254,6 +254,9 @@ class CInstantSendManager : public CRecoveredSigsListener
     mutable Mutex cs_pendingRetry;
     std::unordered_set<uint256, StaticSaltedHasher> pendingRetryTxs GUARDED_BY(cs_pendingRetry);
 
+    mutable Mutex cs_timingsTxSeen;
+    std::unordered_map<uint256, int64_t, StaticSaltedHasher> timingsTxSeen GUARDED_BY(cs_timingsTxSeen);
```

Comment:
it seems as this map can indefinitely grow and eat <s>all</s> significant amount of RAM